Changelog.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## [unreleased]
+- Convert helper methods of module into class methods of model (#6747)
```

Comment:
You should be more specific here and describe exactly what modules and models were involved